robolectric-shadows/shadows-core/src/main/java/org/robolectric/shadows/ShadowAnimation.java

```diff
@@ -1,165 +1,20 @@
- */
-@SuppressWarnings({"UnusedDeclaration"})
 @Implements(Animation.class)
 public class ShadowAnimation {
-
-  private Animation.AnimationListener listener;
-  private Interpolator interpolator;
-  private boolean startFlag = false;
-  private boolean endFlag = false;
-  private long durationMillis = 0;
-  private int repeatCount;
-  private int repeatMode;
-  private long startOffset;
   private int loadedFromResourceId = -1;
 
-  private boolean fillAfter;
-  @RealObject
-  private Animation realAnimation;
-
-  @Implementation
-  public void setAnimationListener(Animation.AnimationListener l) {
-    listener = l;
-  }
-
-  @Implementation
-  public void start() {
-    startFlag = true;
-    if (listener != null) {
-      listener.onAnimationStart(realAnimation);
-    }
-  }
-
-  @Implementation
-  public void cancel() {
-    startFlag = false;
-    endAnimation();
-  }
-
-  @Implementation
-  public boolean hasStarted() {
-    return startFlag;
-  }
-
-  @Implementation
-  public boolean hasEnded() {
-    return endFlag;
-  }
-
-  @Implementation
-  public void setDuration(long durationMillis) {
-    this.durationMillis = durationMillis;
-  }
-
-  @Implementation
-  public long getDuration() {
-    return durationMillis;
-  }
-
-  @Implementation
-  public void setInterpolator(Interpolator interpolator) {
-    this.interpolator = interpolator;
-  }
-
-  @Implementation
-  public Interpolator getInterpolator() {
-    return interpolator;
-  }
-
-  @Implementation
-  public void setRepeatCount(int repeatCount) {
-    this.repeatCount = repeatCount;
-  }
-
-  @Implementation
-  public int getRepeatCount() {
-    return repeatCount;
-  }
-
-  @Implementation
-  public void setRepeatMode(int repeatMode) {
-    this.repeatMode = repeatMode;
-  }
-
-  @Implementation
-  public int getRepeatMode() {
-    return repeatMode;
-  }
-
-  @Implementation
-  public void setStartOffset(long startOffset) {
-    this.startOffset = startOffset;
-  }
-
-  @Implementation
-  public long getStartOffset() {
-    return startOffset;
-  }
-
-  @Implementation
-  public void setFillAfter(boolean fillAfter) {
-    this.fillAfter = fillAfter;
-  }
-
-  @Implementation
-  public boolean getFillAfter() {
-    return fillAfter;
-  }
-
-  /**
-   * Non-Android accessor.
-   *
-   * @return Current animation listener.
-   */
-  public Animation.AnimationListener getAnimationListener() {
-    return listener;
-  }
-
-  /**
-   * Non-Android accessor.  Use to simulate repeat loops of animation.
-   */
-  public void invokeRepeat() {
-    if (listener != null) {
-      listener.onAnimationRepeat(realAnimation);
-    }
-  }
-
-  /**
-   * Non-Android accessor.  Use to simulate end of animation.
-   */
-  public void invokeEnd() {
-    endAnimation();
-    new ShadowAnimationBridge(realAnimation).applyTransformation(1.0f, new Transformation());
-  }
-
   public void setLoadedFromResourceId(int loadedFromResourceId) {
```

Comment:
Am I missing something, but this shadow now only has a single property with setter and gettter, but doesn't shadow any behaviour.... can't it be just deleted now?